QUICKSTART.md

```diff
@@ -0,0 +1,180 @@
+# Quickstart documentation
```

Comment:
- asi by stacilo iba `Quickstart`
- v nadpisom zvykneme pouzivat velke pismena, vid napr.  developers.keboola.com  , https://www.grammarly.com/blog/capitalization-in-the-titles/